### PR TITLE
Update changelog up to 2474e512ce46ae91caa4eabb0dc002e7c2de8e9c

### PR DESCRIPTION
**Pull request by kevingranade**
Holy crap that's a lot, looks like I need to read and summarize something like 3 million words of commits.  Yea it's going to take a while.

### CODE REVIEW

File: data/changelog.txt [hunk 1]
```diff
@@ -1,3 +1,96 @@
...
+Power transmission between vehicles.
+Books need to be read to know what they contain.
+Extend Stamina burn.
+Explosions can create craters.
+Zombies push each other.
+Overhauled shadowcasting to also provide dynamic lighting.
+
+Content:
+Minefield signs
+Lots of improvsed tools (stone hand tools, forge, cooking furniture, clay and pottery).
```
Comment (Night-Pryanik) [inline data/changelog.txt]: Improvised

File: data/changelog.txt [hunk 2]
```diff
@@ -1,3 +1,96 @@
...
+Spawn more items in vehicles.
+Ranch and Ranch-related missions.
+Monster grab and pull attacks.
+Recipes based on offal and corpse waste.
+
+Interface:
+Bionics menu tabbed for better visibility.
+Steamlined reload menu.
+Bundled a square font.
+Seperate zombies better by color.
```
Comment (Night-Pryanik) [inline data/changelog.txt]: Separate

File: data/changelog.txt [hunk 3]
```diff
@@ -1,3 +1,96 @@
...
+Overhauled mapgen to allow more flexible specification of map features.
+Windows build cleanup.
+Better help menu for command line invocation.
+Moved mutation definitions to json.
+3D map infrastructure.
+Move NPC dialogue to json.
+Use lightmap consistently in both curses and tiles.
+More flexible menu filtering system.
+Moved special tool items to json.
+Moved vehicle spwns to json.
```
Comment (Night-Pryanik) [inline data/changelog.txt]: Spawns.

File: data/changelog.txt [hunk 4]
```diff
@@ -1,3 +1,96 @@
...
+More zombies: Elite grenadiers, runners, ferals, predators.
+Buildings.
+Spawn more items in vehicles.
+Ranch and Ranch-related missions.
+Monster grab and pull attacks.
+Recipes based on offal and corpse waste.
+
+Interface:
+Bionics menu tabbed for better visibility.
+Steamlined reload menu.
```
Comment (rm-code) [inline data/changelog.txt]: Streamlined?

File: data/changelog.txt [hunk 5]
```diff
@@ -1,3 +1,96 @@
...
+Lots of dialogs allow use or consumption of items from immediate surroundings as well as inventory.
+Enhancements to AIM.
+
+Mods:
+More Survival Tools
+Mshock Modded Tileset.
+ChestHole tileset covering all entities.
+
+Balance:
+Unify craftng and construction xp gain.
```
Comment (rm-code) [inline data/changelog.txt]: crafting

File: data/changelog.txt [hunk 6]
```diff
@@ -1,3 +1,96 @@
...
+Overhauled encumbrance system for finer degrees of encumbrance.
+
+Bugfixes:
+Fix inability to repair modified clothing.
+Fix charge consumption when invoking tools.
+Fix grenades not exploding.
+Cleanup drivability rules for vehicles.
+Prevent application of traps to players in vehicles.
+Menu crash fixes.
+fix crash when player moves while remotely controlling a vehicle
```
Comment (rm-code) [inline data/changelog.txt]: Fix

**Comment by Leland** [comment]
itshappening.gif

**Comment by Leland** [comment]
itshappening.gif

If you see to parcel out the work, I'd be more than happy to assist. 3 million words is, uh, a lot.

**Comment by kevingranade** [comment]
Feel free to make changelog PRs, I do this
```git log 2474e512ce46ae91caa4eabb0dc002e7c2de8e9c..```
(Note the two periods at the end)
To get a list of commits that haven't been summarized yet, then record the last commit summarized as part of the changelog commit.

**Comment by kevingranade** [comment]
Also, I was planning on doing some jq to find the number of added json entities for the stats section, if someone can do that and PR it that'd speed things up.

File: data/changelog.txt [hunk 7]
```diff
@@ -1,3 +1,96 @@
...
+Windows build cleanup.
+Better help menu for command line invocation.
+Moved mutation definitions to json.
+3D map infrastructure.
+Move NPC dialogue to json.
+Use lightmap consistently in both curses and tiles.
+More flexible menu filtering system.
+Moved special tool items to json.
+Moved vehicle spawns to json.
+Methods to transform buildings after generation, i.e. to make them looted or barricaded.
```
Comment (ZhilkinSerg) [inline data/changelog.txt]: What's the PR number for this?

**Comment by Leland** [comment]
@kevingranade for reference here's a slightly quicker way to grab that git log in reverse format:

`git log --reverse --ancestry-path <old-commit>..`

This way you won't have to scroll to the oldest commit

**Pull request merged**